addOns/encoder/CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Maintenance changes.
 - Allow script processors to return strings without requiring an "EncodeDecodeResult" wrapper.
 - Show help and options buttons in the main dialog.
+- The Base64 decoder is now slightly more robust and handles line wrapped input.
```

Comment:
It could be more explicit that it's using MIME variant. Worth mentioning in the help (maybe link to the JavaDoc which has the detailed description).